builtin/providers/datadog/resource_datadog_timeboard.go

```diff
@@ -373,9 +371,9 @@ func appendRequests(datadogGraph *datadog.Graph, terraformRequests *[]interface{
 func appendEvents(datadogGraph *datadog.Graph, terraformEvents *[]interface{}) {
 	for _, t_ := range *terraformEvents {
```

Comment:
One of the Graph nested structs that would be nicer to use if it would be broken apart. If raised https://github.com/zorkian/go-datadog-api/issues/98 to do this.